lib/core/versions/0.4.0/OperationProcessor.ts

```diff
@@ -20,81 +20,78 @@ export default class OperationProcessor implements IOperationProcessor {
     anchoredOperationModel: AnchoredOperationModel,
     didDocumentReference: { didDocument: object | undefined }
   ): Promise<ApplyResult> {
-    let operationHash = undefined;
 
     try {
       // NOTE: only used for read interally.
       const didDocument = didDocumentReference.didDocument as (DocumentModel | undefined);
-
       const operation = AnchoredOperation.createAnchoredOperation(anchoredOperationModel);
-      operationHash = operation.operationHash;
 
       if (operation.type === OperationType.Create) {
 
         // If we have seen a previous create operation.
         if (didDocumentReference.didDocument) {
-          return { validOperation: false, operationHash };
+          return { validOperation: false };
```

Comment:
Why are we modifying files from previous versions?

Reply:
I have the same question.

Reply:
We normally should avoid modifying code in previous versions as much as possible. But since this change involves an interface change (which in theory should really be minimized, but here we are), so changing old code to match the interface.